server/models/Profile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+const userSchema = require("./User");
+const email = User.find_by_id({});
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  dob: {
+    type: Date,
+    required: true,
+  },
+  user: {
```

Comment:
Wouldn't it be better to add a reference to the User model here instead and maybe call this field `userId`? :)

Reply:
Do you mean something like this.
{
    type: mongoose.Schema.Types.ObjectId,
    ref: "user",
    required: true,
}